src/LEGO.AsyncAPI.Bindings/Sns/FilterPolicy.cs

```diff
@@ -0,0 +1,26 @@
+namespace LEGO.AsyncAPI.Bindings.Sns
+{
+    using System;
+    using LEGO.AsyncAPI.Models.Interfaces;
+    using LEGO.AsyncAPI.Writers;
+
+    public class FilterPolicy : IAsyncApiElement
+    {
+        /// <summary>
+        /// A map of a message attribute to an array of possible matches. The match may be a simple string for an exact match, but it may also be an object that represents a constraint and values for that constraint.
+        /// </summary>
+        public IAsyncApiAny Attributes { get; set; }
+
+        public void Serialize(IAsyncApiWriter writer)
+        {
+            if (writer is null)
+            {
+                throw new ArgumentNullException(nameof(writer));
+            }
+
+            writer.WriteStartObject();
+            writer.WriteRequiredObject("attributes", this.Attributes, (w, a) => a.Write(w));
```

Comment:
```suggestion
            writer.WriteRequiredObject("attributes", this.Attributes, (w, a) => a.WriteAny(w));
```

Reply:
When I try this it says that `writeAny` doesn't exist as a function on it 🤔

Reply:
It's in the `AsyncApiWriterAnyExtensions` class. Should definitely be public.

Reply:
Mind you its
<img width="851" alt="image" src="https://github.com/LEGO/AsyncAPI.NET/assets/5294032/0cd4230b-79b7-4b28-8039-90c7e35ee505">

It looks like you have `node.WriteAny(writer)` its `Writer.WriteAny(node)`

Reply:
Right, got it. Will give it a go! Thanks again for all your continued support on this. Really appreciate it! I think we're in the home stretch now 😄